### ISSUE TITLE
Assign different colors to different Stategies / Selektor

### ISSUE THREAD

**Issue by rukemn** [jotoh98/treasureHunt#239]
Right now each Strategy / Selector is using an arbitrary color.
This leads to confusion for the user, since he cannot distinguish between each of the actors.

Instead I suggest assigning each of them a "base color" , such that they can use different shades of that color. 

e.g.
Selektor : White -ish
Searcher: Green - ish
Hider: Red/Orange -ish

One way of doing this could be to predefine 10 or so shades for each base color via the prefences.

**Comment by rank-and-file** [comment]
Good idea, I think the suggested colors would be appropriate, maybe Searcher could be Green/Blue -ish
